modules/core/src/main/scala/org/tessellation/infrastructure/trust/storage/TrustStorage.scala

```diff
@@ -78,11 +89,37 @@ object TrustStorage {
 
       def getTrust: F[TrustMap] = trustStoreRef.get.map(_.trust)
 
+      def getBiasedTrust: F[TrustMap] = getTrust.map { trust =>
+        seedlist.fold(trust) { peerIds =>
+          val updatedTrust = peerIds
+            .filterNot(trust.trust.contains)
+            .map(_ -> TrustInfo(predictedTrust = config.seedlistInputBias.some))
+            .toMap
+          val updatedPeerLabels = peerIds
+            .filterNot(trust.peerLabels.value.contains)
+            .map(_ -> PublicTrust.empty)
+            .toMap
+          TrustMap(
+            trust.trust ++ updatedTrust,
+            PublicTrustMap(trust.peerLabels.value ++ updatedPeerLabels)
+          )
+        }
+      }
+
       def getCurrentOrdinalTrust: F[OrdinalTrustMap] = trustStoreRef.get.map(_.currentOrdinalTrust)
 
       def getNextOrdinalTrust: F[OrdinalTrustMap] = trustStoreRef.get.map(_.nextOrdinalTrust)
 
-      def getPublicTrust: F[PublicTrust] = getTrust.map(_.toPublicTrust)
+      def getPublicTrust: F[PublicTrust] = getTrust
+        .map(_.toPublicTrust)
+        .map(
+          _.focus(_.labels)
+            .modify(
+              _.view
+                .mapValues(max(config.seedlistOutputBias, _))
```

Comment:
So whenever the labels get below `seedListOutputBias` you want them to default back to `seedlistOutputBias` value? Wouldn't it be better to have the logic set it when the labels are calculated rather than let them go below and then override them here?

Reply:
It's better to store the original model outputs and only take this as a view into them so we're not overwriting values.